src/main/java/com/aws/greengrass/clientdevices/auth/iot/usecases/VerifyIotCertificate.java

```diff
@@ -46,7 +45,7 @@ public VerifyIotCertificate(IotAuthClient iotAuthClient, CertificateRegistry cer
     }
 
     @Override
-    public Result<Boolean> apply(String certificatePem) {
+    public Boolean apply(String certificatePem) {
```

Comment:
nit: can use `boolean` instead of `Boolean`, which is an object wrapping the native type

Reply:
I can't the Generic type doesn't allow me to use primitives there